src/webhook/setTestTopicAutoMessage.js

```diff
@@ -0,0 +1,25 @@
+const {registerV2Queue, followTopicQueue} = require('../config');
+
+const setTestTopicAutoMessage = async (req, res) => {
+  try {
+    /**
+     * @type {RegisterV2ProcessBody}
+     */
+    const body = req?.body;
+    const result = await followTopicQueue.add(body);
+
+    return res.json({
+      status: 'ok',
+      result
+    });
+  } catch (error) {
+    console.error('error');
+    console.error(error);
+    return res.json({
+      ststus: 'error',
+      message: error
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the error response object key 'status'. Also, directly returning the error object in the response could potentially expose sensitive information. It would be better to return a generic error message to the client and log the actual error for debugging purposes.
```diff
-      ststus: 'error',
-      message: error
+      status: 'error',
+      message: 'An error occurred while processing your request.'
```
Remember to log the error using a logging library or service like Winston or Sentry, which can provide more context about the error, such as timestamps, stack traces, and server statuses.



<!-- This is an auto-generated comment by OSS CodeRabbit -->